src/main/kotlin/org/rucca/cheese/model/NotificationsUnreadCountGet200ResponseDTO.kt

```
@@ -0,0 +1,23 @@
+data class NotificationsUnreadCountGet200ResponseDTO(
+    @Schema(example = "null", required = true, description = "")
+    @get:JsonProperty("code", required = true)
+    val code: kotlin.Int = 0,
+    @Schema(example = "null", required = true, description = "")
+    @get:JsonProperty("message", required = true)
+    val message: kotlin.String = "success",
+    @field:Valid
+    @Schema(example = "null", required = true, description = "")
+    @get:JsonProperty("data", required = true)
+    val `data`: NotificationsUnreadCountGet200ResponseDataDTO
+) {}
```

detekt (1.23.7)

[warning] The class or object NotificationsUnreadCountGet200ResponseDTO is empty.

(detekt.empty-blocks.EmptyClassBlock)